include/rosconsole_bridge/bridge.h

```diff
@@ -46,6 +46,7 @@ class OutputHandlerROS : public console_bridge::OutputHandler
 {
 public:
   OutputHandlerROS(void);
+  ~OutputHandlerROS();
```

Comment:
There is an interesting talk about ABI compatibility: 
https://events.linuxfoundation.org/sites/events/files/slides/Binary_Compatibility_for_library_devs.pdf

Essentially, because the v-table isn't changed, ABI compatibility is maintained.
This is verified by the [ABI compatibility checker](http://ispras.linuxbase.org/index.php/ABI_compliance_checker) - an interesting tool, you should have a look at:
```
$ abi-compliance-checker -lib rosconsole_bridge -old /tmp/old.xml -new /tmp/new.xml
preparation, please wait ...
Using GCC 4.8 (x86_64-linux-gnu)
checking header(s) old ...
checking header(s) new ...
comparing ABIs ...
comparing APIs ...
creating compatibility report ...
result: COMPATIBLE
total "Binary" compatibility problems: 0, warnings: 0
total "Source" compatibility problems: 0, warnings: 0
```